pkg/util/tests/mockgenerator/client/client.go

```diff
@@ -1,4 +1,4 @@
 package client
 
-//go:generate mockgen -destination ../../generated/mocks/status-writer.go -package mocks sigs.k8s.io/controller-runtime/pkg/client StatusWriter
-//go:generate mockgen -destination ../../generated/mocks/cr-client.go -package mocks sigs.k8s.io/controller-runtime/pkg/client Client
+//go:generate mockgen -destination ../../generated/mocks/client/status-writer.go -package $GOPACKAGE sigs.k8s.io/controller-runtime/pkg/client StatusWriter
```

Comment:
What's the point in putting those commands here rather than the Makefile if we have a separate package just to list those commands?

Reply:
I didn't want to change the makefile too much, and this means we generate based on files we have.
can go either way on it (keep here/move to makefile)

Reply:
@RiRa12621 what is your opinion on the matter?

Reply:
+1 for keeping the makefile clean